docs/content/preview/explore/fault-tolerance/macos.md

```diff
@@ -96,35 +64,39 @@ $ java -jar ./yb-sample-apps.jar --workload SqlInserts \
                                  --num_threads_read 4
 ```
 
-The `SqlInserts` workload prints some statistics while running, which is also shown below. You can read more details about the output of the workload applications at the [YugabyteDB workload generator](https://github.com/yugabyte/yb-sample-apps).
+The `SqlInserts` workload prints some statistics while running as follows. You can read more details about the output of the workload applications at the [YugabyteDB workload generator](https://github.com/yugabyte/yb-sample-apps).
```

Comment:
The `SqlInserts` workload prints some statistics while running as follows:

```output
32001 [Thread-1] INFO com.yugabyte.sample.common.metrics.MetricsTracker  - Read: 4508.59 ops/sec (0.88 ms/op), 121328 total ops  |  Write: 658.11 ops/sec (1.51 ms/op), 18154 total ops  |  Uptime: 30024 ms | ...
37006 [Thread-1] INFO com.yugabyte.sample.common.metrics.MetricsTracker  - Read: 4342.41 ops/sec (0.92 ms/op), 143061 total ops  |  Write: 635.59 ops/sec (1.58 ms/op), 21335 total ops  |  Uptime: 35029 ms | ...
```
For more information about the output of the workload applications, see [YugabyteDB workload generator](https://github.com/yugabyte/yb-sample-apps).